prov/verbs/src/fi_verbs.c

```diff
@@ -990,14 +1004,8 @@ fi_ibv_create_ep(const char *node, const char *service,
 
 	ret = rdma_create_ep(id, _rai, NULL, NULL);
 	if (ret) {
+		VERBS_INFO_ENO(FABRIC, "rdma_create_ep", errno);
 		ret = -errno;
-		if (ret == -ENOENT) {
-			FI_WARN(&fi_ibv_prov, FI_LOG_CORE,
-				"rdma_create_ep()-->ENOENT; "
-				"likely usnic bug, skipping verbs provider.\n");
-			ret = -FI_ENODATA;
-		}
-		FI_INFO(&fi_ibv_prov, FI_LOG_CORE, "Unable to create rdma_cm id\n");
```

Comment:
OK, nevermind.  This bit of logic is no longer needed now that Sean included #988 / f89e49e.  This particular bit of logic would probably still help on a machine where a valid verbs device (e.g., a Mellanox HCA or an Intel iWarp card) is present alongside a Cisco VIC with usNIC enabled, but that case is going to be pretty rare in practice.  Also, the situation can be worked around by specifying a provider to `fi_getinfo` or via the `FI_PROVIDER` environment variable.